tools/packaging/kernel/build-kernel.sh

```diff
@@ -87,6 +87,7 @@ Options:
 	-c <path>   	: Path to config file to build the kernel.
 	-d          	: Enable bash debug.
 	-e          	: Enable experimental kernel.
+	-E		: Enable arch-specific experimental kernel, arch info offered by "-a".
```

Comment:
Nit: Can you indent with spaces here? LGTM otherwise.

Reply:
thanks @Jakob-Naucke, done.

Reply:
I'm sorry, but it's indented too far.

Reply:
what about this?

Reply:
Thanks @jongwu, I'll TAL at the rest again... (btw, I said "indent with spaces", that was incorrect, we're indenting with tabs here. I meant "align with spaces". Sorry!)